modules/apps/frontend-data-set/frontend-data-set-web/src/main/resources/META-INF/resources/views/list/List.js

```diff
@@ -109,9 +121,7 @@ const ListItem = ({item, schema}) => {
 			)}
 
 			<ClayList.ItemField className="justify-content-center" expand>
-				{title && (
-					<ClayList.ItemTitle>{item[title]}</ClayList.ItemTitle>
-				)}
+				{renderTitle(item)}
```

Comment:
This works fine, but a more React-y way of doing this would be to add a new component, something like

```js
const Title = ({item}) => {
    ...
}
```

and then do
```js
<ClayList.ItemField className="justify-content-center" expand>
    <Title item={item}>
```

Reply:
Good point! I'll go with your suggested change!